src/Arcus.Messaging.ServiceBus.Core/Extensions/MessageExtensions.cs

```diff
@@ -31,19 +29,5 @@ public static T GetUserProperty<T>(this Message message, string key)
             throw new KeyNotFoundException(
                 $"No user property with the key: '{key}' was found in the Service Bus message");
         }
-
-        /// <summary>
-        ///     Gets the transaction id that is linked to this message
-        /// </summary>
-        /// <param name="message">Message to process</param>
-        /// <returns>Transaction id for message</returns>
-        public static string GetTransactionId(this Message message)
```

Comment:
Do we have to remove this? You could keep this around and base it off of `GetMessageCorrelationInfo`?